docker/env/hydra.sh

```diff
@@ -173,6 +173,9 @@ AWS_OPTIONS=$(env | grep AWS_ | cut -d "=" -f 1 | xargs -i echo "--env {}")
 # export all JENKINS_* env vars into the docker run
 JENKINS_OPTIONS=$(env | grep JENKINS_ | cut -d "=" -f 1 | xargs -i echo "--env {}")
 
+# export all GIT_* env vars into the docker run
+GIT_OPTIONS=$(env | grep GIT_ | cut -d "=" -f 1 | xargs -i echo "--env {}")
```

Comment:
I don't think we want to pass all git env vars to the hydra. We already do this for `GIT_USER_EMAIL`, so just add `-e GIT_BRANCH` as a line after.

Reply:
Fixed.